theme/debugger.less

```diff
@@ -63,18 +63,34 @@
     }
 
     .variable {
-        color: white;
+        // color: white;
+    }
+    .transparent{
+        opacity: 0;
     }
     .variable.changed {
-        outline: solid 2px yellow;
+        // outline: solid 2px yellow;
     }
     .variable .varname {
         font-weight: bold;
     }
     .variable .detail {
-        background:white;
+        // background:white;
+        // color: #87D282;
         color:black;
     }
+    .varval.number {
+        color: darkorchid; // TODO: Change this for the color we want for numbers
```

Comment:
define LESS variables in "site.variables" and use those

Reply:
or define them at the top of the file, and have them reference standard lib colors in site.variables

Reply:
but always use less variables.